.readthedocs.yaml

```diff
@@ -13,6 +13,9 @@ sphinx:
 
 # Set the version of Python and other tools you might need
 build:
+  # Install Graphviz to build SVG files
+  apt_packages:
+    - "graphviz"
```

Comment:
Will the documentation build fail if this is not installed? If so, we might want to add this to "Install from source" documentation.

Reply:
The build will not fail, but on a previous build where it wasn't installed, it did not generate the graphs from the `.. digraph::` directive—showing this warning message:

```
"readthedocs" WARNING: dot command 'dot' cannot be run (needed for graphviz output), check the graphviz_dot setting
```

Reply:
We had a mention of `graphviz` in the [Developer installation instructions](https://docs.pybamm.org/en/latest/source/user_guide/installation/GNU-linux.html#developer-install), adding it to the source installation instructions too